tests/foreman/api/test_syncplan.py

```diff
@@ -295,10 +320,22 @@ def test_remove_one_product_to_sync_plan(self):
         @Feature: Sync Plan
 
         """
+        syncplan_id = entities.SyncPlan(
+            organization=self.org_id
+        ).create_json()['id']
+        product_ids = tuple(
+            entities.Product(organization=self.org_id).create_json()['id']
+            for _ in range(2)
+        )
+        syncplan = entities.SyncPlan(id=syncplan_id, organization=self.org_id)
+
+        syncplan.add_products(product_ids)
+        self.assertEqual(len(syncplan.read_json()['products']), 2)
+        syncplan.remove_products([product_ids[0]])
+        self.assertEqual(len(syncplan.read_json()['products']), 1)
```

Comment:
Would it be too much to ask that you assert that the product 'left' is the one you did not delete? Just to be safe?